lib/versioning/nuget/index.spec.ts

```diff
@@ -1,60 +1,181 @@
 import nuget from '.';
```

Comment:
@viceice @rarkins Before I proceed further, can you please take a look at these tests? Everything is correct?